website/src/docs/hotchocolate/v12/api-reference/apollo-federation.md

```diff
@@ -1,7 +1,295 @@
 ---
 title: Apollo Federation Subgraph Support
 ---
+> If you want to read more about Apollo Federation in general, you can head over to the [Apollo Federation documentation](https://www.apollographql.com/docs/federation/), which provides a robust overview and set of examples for this GraphQL architectural pattern. Many of the core principles and concepts are referenced within this document.
 
-> Note: Apollo Federation Support is coming with Hot Chocolate 12.6
+Hot Chocolate includes an implementation of the Apollo Federation v2 specification for creating Apollo Federated subgraphs. Through Apollo Federation, you can combine multiple GraphQL APIs into a single API for your consumers.
 
-## Example subgraphs
+The documentation describes the syntax for creating an Apollo Federated subgraph using Hot Chocolate and relates the implementation specifics to its counterpart in the Apollo Federation docs. This document *will not* provide a thorough explanation of the Apollo Federation core concepts nor will it describe how you go about creating a supergraph to stitch together various subgraphs, as the Apollo Federation team already provides thorough documentation of those principles.
```

Comment:
I think afterwards we could link to our Apollo Federation examples for folks that just want to look at a working setup: https://github.com/ChilliCream/graphql-platform/tree/main/src/HotChocolate/ApolloFederation/examples

Reply:
Love the suggestion! I think an additional contribution I'd like to make, if I can make the time, is setting up one or both of the solutions with a `docker-compose.yaml` file that would allow someone to easily spin up all of the subgraphs in one easy command. Pair that with the fact that Visual Studio has support for `docker-compose.yaml` and we can provide users with an easy "click once" style debugging session that can help them interact with the example and see the pieces working all together 👍